openshift/generate-manifests.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Know where the repo root is so we can reference things relative to it
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# Source bingo so we can use kustomize and yq
+. "${REPO_ROOT}/.bingo/variables.env"
+
+# We're going to do file manipulation, so let's work in a temp dir
+TMP_ROOT="$(mktemp -p . -d 2>/dev/null || mktemp -d ./tmpdir.XXXXXXX)"
+# Make sure to delete the temp dir when we exit
+trap 'rm -rf $TMP_ROOT' EXIT
+
+# Copy all kustomize files into a temp dir
+TMP_CONFIG="${TMP_ROOT}/manifests"
+cp -a "${REPO_ROOT}/manifests" "$TMP_CONFIG"
+
+# Override core namespace to openshift-rukpak
+$YQ -i '.namespace = "openshift-rukpak"' "${TMP_CONFIG}/core/kustomization.yaml"
+# Override helm-provisioner namespace to openshift-rukpak
+$YQ -i '.namespace = "openshift-rukpak"' "${TMP_CONFIG}/provisioners/kustomization.yaml"
+# Override apis namespace to openshift-rukpak
+$YQ -i '.namespace = "openshift-rukpak"' "${TMP_CONFIG}/apis/kustomization.yml"
+# Override crd-validator namespace to openshift-crd-validator
+$YQ -i '.namespace = "openshift-crd-validator"' "${TMP_CONFIG}/crdvalidator/kustomization.yml"
```

Comment:
You should go ahead and templatize the namespaces and images so we can replace them in cluster-olm-operator. 
Something like https://github.com/openshift/operator-framework-operator-controller/commit/f8bd9cea4551f467b1d621cbfc081d56f6baeace

Reply:
Sorry - this was actually outdated and removed when the manifests got merged in. This is all managed by a `namespace` declaration inside `openshift/kustomization.yaml` now. Should be easily replaceable with `kubectl kustomize edit set namespace` or I could put it in the script, WDYT?